examples/llm/README.md

```diff
@@ -0,0 +1,2 @@
+# Use Hamilton to power your LLM application [WIP]
```

Comment:
A general file overview would be helpful, and could you please add a visualization of what we've defined in Hamilton, using the .visualize functionality of the driver, and add that as a line to show people how they can visualize their code.

Reply:
Untangling the 2 branches.